runtime/src/accounts_background_service.rs

```diff
@@ -120,7 +122,7 @@ impl SnapshotRequestHandler {
                 shrink_time.stop();
 
                 let mut flush_accounts_cache_time = Measure::start("flush_accounts_cache_time");
-                if accounts_db_caching_enabled {
+                if accounts_db_caching_enabled && !just_clean_and_shrink {
                     // Forced cache flushing MUST flush all roots <= snapshot_root_bank.slot().
                     // That's because `snapshot_root_bank.slot()` must be root at this point,
                     // and contains relevant updates because each bank has at least 1 account update due
```

Comment:
@carllin line 134 below fails when I try to kick off a clean earlier. Clearly something isn't quite happy. I haven't had a chance to dig into this failure yet. So, for now, I disabled the cache flush. That may not be correct depending on if we've been doing local replay and there are contents in the cache.